client/package.json

```diff
@@ -14,7 +14,8 @@
     "lint": "yarn lint:flow && yarn lint:eslint",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "start": "yarn storybook"
+    "start": "yarn storybook",
+    "heroku-postbuild": "npm rebuild node-sass"
```

Comment:
Would you call `heroku-postbuild` in the CircleCI config?

Reply:
Lets use @baohouse  PR. I will close this as they both solve the same problem.

I have my own CIrcleCI to get us on CircleCI 2.0, but its breaking the PR process for my branches sadly. 

So I will close this in favor of @baohouse 's working branch.